### PR TITLE
Feature: shifts default sensitive props values into pillar data

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: salt/pillar/data-pipeline.sls [hunk 1]
```diff
@@ -11,6 +11,11 @@ data_pipeline:
             client_id: dummy-client-id
             client_secret: dummy-client-secret
 
+        sensitive_props:
+            key: ""
```
Comment (giorgiosironi) [inline salt/pillar/data-pipeline.sls:15]: only `key` is actually sensitive in here, the rest is like saying `TLS 1.1`; but they are all nicely grouped together and should be in pillars. Would just called them `cryptography_props` maybe?

Comment (lsh-0) [reply salt/pillar/data-pipeline.sls:15]: they're called `sensitive-props` in the nifi configuration file, I'll stick with that for now.

from the documentation I'm under the impression that the values are used together to perform the encryption, with the `key` property being the only *really* sensitive value.

**Comment by lsh-0** [comment]
failures are all cascading failures from the mirror originally used to download nifi now broken.

merging

File: salt/pillar/data-pipeline.sls (hunk 1 above)
Comment (giorgiosironi) [reply salt/pillar/data-pipeline.sls:15]: Good if it's a terminology coming from Nifi

**Pull request merged**